src/Atmos/Model/turbulence.jl

```diff
@@ -253,20 +275,29 @@ url = {https://link.aps.org/doi/10.1103/PhysRevFluids.1.041701}
 struct AnisoMinDiss{FT} <: TurbulenceClosure
   C_poincare::FT
 end
-vars_aux(::AnisoMinDiss,T) = @vars(Δ::T)
-vars_gradient(::AnisoMinDiss,T) = @vars(θ_v::T)
+vars_aux(::AnisoMinDiss,FT) = @vars(Δ::FT,ρν::FT,BR::FT,Freq::FT)
+vars_gradient(::AnisoMinDiss,FT) = @vars(θ_v::FT)
 function atmos_init_aux!(::AnisoMinDiss, ::AtmosModel, aux::Vars, geom::LocalGeometry)
   aux.turbulence.Δ = lengthscale(geom)
+  aux.turbulence.ρν = 0
+  aux.turbulence.BR = 0
+end
+function atmos_nodal_update_aux!(::AnisoMinDiss, ::AtmosModel, state::Vars, aux::Vars, diffusive::Vars, t::Real)
+  aux.turbulence.ρν = diffusive.ρν
+  aux.turbulence.BR = diffusive.turbulence.BR
```

Comment:
where are these used?

Reply:
Just searched. It's used in diagnostics, but then the variable that is assigned to it is never used, so we can remove it.

Reply:
Actually..... It _is_ used in [diagnostics](https://github.com/climate-machine/CLIMA/pull/610/files#diff-f26cd0b9cbdc8868d486fccf30ef3d2aR260-R261), which is subsequently called [here](https://github.com/climate-machine/CLIMA/pull/610/files#diff-8210716d7cccb7f5fcbb29c7b43aae59R69).

Reply:
To avoid wasting significant resources, we should compute this on the fly. But computing on the fly leads to the possibility of inconsistent formulas.

We need to incorporate `AtmosModel` into the diagnostics to have consistent flow control of computed terms. Otherwise, we will continue to deal with the possibility of inconsistent formulas.

Reply:
Right, they are quantities that are written out to file for post-processing. 
They may be not necessary operationally, but we certainly need them while debugging our turbulence models.

Reply:
@simonbyrne up to you whether you want to leave this there for diagnostics, or remove and update the diagnostics later.

Reply:
These should be implemented in diagnostics.